src/marine/glider2ioda.py

```diff
@@ -104,15 +117,28 @@ def main():
         '-d', '--date',
         help="base date for the center of the window",
         metavar="YYYYMMDDHH", type=str, required=True)
+    required.add_argument(
+        '-t', '--thin',
+        help="1 to thin; 0 not to thin",
+        type=float, default=0.0)
```

Comment:
Might make the intention more clear to change this to `type=bool`, which can be either `True` or `False`.